client/src/app/appCtrl.js

```diff
@@ -257,9 +253,10 @@ angular.module('app').controller('appCtrl', ['$scope', '$timeout', 'appConfig',
   // return: matrix with numeric columns
   // TIMESTAMP is always the 1st column.
   var generateFieldMap = function(row, excludes) {
+    var usedTimestamp = appConfig.TIMESTAMP;
     if (!row.hasOwnProperty(appConfig.TIMESTAMP)) {
-      handleError("No timestamp field was found", "warning");
-      return null;
+      handleError("No timestamp field was found, fallback to iterations", "warning");
```

Comment:
Suggested re-wordings: 
"No timestamp field was found, falling back to iterations"
or:
"No timestamp field was found, using iterations instead"

Also, suggest we use "info" instead of "warning" here, as it is not really a warning.